test/shared_array_scope_list.rb

```diff
@@ -78,6 +83,11 @@ def test_insert_at
       new = ArrayScopeListMixin.create(parent_id: 20, parent_type: 'ParentClass')
       assert_equal 2, new.pos
 
+      new = ArrayScopeListMixin.act_as_list_no_update do
+        ArrayScopeListMixin.create(parent_id: 20, parent_type: 'ParentClass')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.